### ISSUE TITLE
Put patch-level labels in a Dataloader, ensure it loads in notebook

### ISSUE THREAD

**Issue by soniajoseph** [soniajoseph/ViT-Prisma#84]
ImageNet labels are way too coarse-grained. @themachinefan put ImageNet through a SAM pipeline to get a label for each patch.

The results are here: https://huggingface.co/datasets/Prisma-Multimodal/segmented-imagenet1k-subset

We need to make sure these results load cleanly into a Dataloader, which we can cleanly query to get the label per patch

Some things to keep in mind are:
* The DataLoader should keep in mind that patch size is a hyperparameter that can vary. Patch size usually 8, 16, or 32.
* The above labels currently are in the form of a boolean mask. Could we make this code more efficient? 
* Ideally, there is a function that takes the the patch number, and returns the labels for that patch returned as a list of strings.

**Comment by jain18ayush** [comment]
picking up this issue

**Comment by suchir-madap** [comment]
Picking up this issue!
